Bebb-Protocol-and-Apps/PWS
# Delete user space

Allows users to "delete" Spaces
As Spaces are NFTs, they are not to be deleted. We hide them instead, such that the Space cannot be retrieved anymore. It's still available as an NFT though (e.g. if users look into their wallet).

To test:
On the My Spaces tab, find the Delete button for each Space. 
Follow the Delete process.
The Deletion should be successful; the Space doesn't show up anymore.
There should not be any side-effects on other Spaces and functionality.
